NicolaeChristof/Descend-Into-Darkness
# Feature ladder

This PR includes the functional ladders and the lock and door.

How to Test:

Ladder:
- in the test level below the breakable floor is a ladder. Approach it and press E, you should now be able to move up and down freely. Press E again to get off the ladder or move away from it. (I plan to make this a bit smoother but it is functional for now)

Locked door:
- in the Mine level there are two keys (bright purple balls) and two locked doors (big white wall). Each key corresponds to a specific door. Press E on the door after collecting the correct key, the door will disappear (this will obviously change when we get more assets)

Known Bugs:
- No known bugs
